api/types/swarm/swarm.go

```diff
@@ -159,6 +160,7 @@ type InitRequest struct {
 	Availability     NodeAvailability
 	DefaultAddrPool  []string
 	SubnetSize       uint32
+	DataPathPort     uint32
```

Comment:
Sorry, forgot to mention that earlier; can you move this below `DataPathAddr`, so that they're next to each other?